tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compileOnSave": false,
   "compilerOptions": {
     "importHelpers": true,
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
is there something wrong with esnext? as clarity also compiles that way, i would recommend keeping it

Reply:
`ng update` did this

Reply:
https://angular.io/guide/migration-update-module-and-target-compiler-options